.gitignore

```diff
@@ -4,4 +4,4 @@ bundle.tar.gz
 dist/
 cover.out
 tmp/
-
+.idea/
```

Comment:
Couldn't be happier either too 😄  Would it be better to add it to your personal global .gitignore though instead of having IDE specifics here? (https://sourabhbajaj.com/mac-setup/Git/gitignore.html)